cms/envs/test.py

```diff
@@ -129,9 +130,11 @@
     },
 }
 
-# This hack disables migrations during tests. We want to create tables directly from the models for speed.
-# See https://groups.google.com/d/msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ.
-MIGRATION_MODULES = {app: "app.migrations_not_used_in_tests" for app in INSTALLED_APPS}
+
+# Disables migrations and creates tables directly from models for speed.
+os.environ['ENABLE_MIGRATIONS'] = '1'
```

Comment:
I'm confused by this logic. Why is it setting ENABLE_MIGRATIONS to 1, when the comment says that it is disabling migrations? Doesn't the test then always fail because the setting is true?